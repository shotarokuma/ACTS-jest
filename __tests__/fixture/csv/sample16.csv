
true	true	61	true	true	false	true	74	false	false