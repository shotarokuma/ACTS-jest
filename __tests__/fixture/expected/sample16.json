[
{"0":"true","1":"true","2":"61","3":"true","4":"true","5":"false","6":"true","7":"74","8":"false","9":"false"}
]
